docs/diginstroom/sip/2.0/profiles/film.md

```diff
@@ -0,0 +1,251 @@
+---
+layout:       default
+title:        Film
+parent:       Profiles
+grand_parent:  2.0
+nav_order:    4
+nav_exclude:  false
+---
+Editor's Draft
+{: .label .label-yellow }
+# Profile: Film 
+
+The film profile supports the ingest of digitised film consisting of one or more image and/or audio reels. 
+This profile dictates how the media files (in file formats such as ZIP, MOV and DPX), their metadata, and the relationships between them, should be expressed and organized.
+
+It mainly applies the [DCTERMS metadata schema](https://www.dublincore.org/schemas/xmls/qdc/dcterms.xsd) for descriptive metadata and allows extensions using [Schema.org](https://schema.org), thereby resembling the [Material Artwork profile](https://data.hetarchief.be/id/sip/2.0/material-artwork). 
+
+Its additions lie in the introduction of a separate representation to denote the physical carrier(s) (a so-called 'carrier representation') and custom film-specific metadata (using `<premis:significantProperties>` elements in the package PREMIS file) to describe physical aspects of this/these carrier(s).
+This carrier representation was added to facilitate the description of the physical carrier(s), since the PREMIS metadata schema itself doesn't offer this possibility. 
+
+**Permalink:** <https://data.hetarchief.be/id/sip/2.0/film>
+
+## Example Directory structure
+
+```plaintext
+root_directory
+├── METS.xml
+├── metadata
+│   ├── descriptive
+│   │   └── dc+schema.xml
+│   └── preservation
+│       └── premis.xml                                  # package PREMIS
+│
+└── representations
+    │
+    ├── representation_1
+    │   ├── METS.xml
+    │   ├── data
+    │   │   └── master.zip                              # ZIP containing DPX files
+    │   └── metadata
+    │       └── preservation
+    │           └── premis.xml
+    │
+    ├── representation_2
+    │   ├── METS.xml
+    │   ├── data
+    │   │   └── mezzanine.mov                           # Playable mezzanine
+    │   └── metadata
+    │       └── preservation
+    │           └── premis.xml
+    │
+    └── representation_3
+        ├── METS.xml
+        ├── data
+        │   └── QC.dpx                                  # DPX for QC
+        └── metadata
+            └── preservation
+                └── premis.xml
+```
+
+## Requirements
+
+### General
+
+- A SIP MUST contain content of exactly one digitised film, consisting of one or more image and/or audio reels.
+- <mark> The content MUST be digitised per reel, i.e. each ZIP and/or MOV and/or DPX file contained in their respective representation directories MUST represent exactly one film consisting of one or more image and/or audio reels.</mark>
```

Comment:
I this is correct (check with team digi), but the phrasing is a little weird: we're not steering digitization here, so the sentence before  i.e. is irrelevant in this context and the last part says enough.

I don't know if the following suggestion is still correct:
```suggestion
- Each ZIP, MOV, or DPX file contained in their respective representation directories MUST represent exactly one image or audio reel.
```